### ISSUE TITLE
Keyboard navigation on power search

### ISSUE THREAD

**Issue by liyasthomas** [liyasthomas/postwoman#1879]
I'm always frustrated when the keyboard navigation doesn't work on lists. Power search has an auto-complete / search suggestions list that doesn't support keyboard navigation.

To reproduce:
1. Press <kbd>/</kbd> from any page in [Hoppscotch](https://hoppscotch.io) to bring up power search
2. Try navigating through search suggestions or auto-complete if you type in a query through the keyboard: <kbd>↑</kbd> or <kbd>↓</kbd>
3. <kbd>↑</kbd> or <kbd>↓</kbd> keyboard Navigation doesn't work

<img width="1440" alt="Screenshot 2021-10-17 at 11 41 45 AM" src="https://user-images.githubusercontent.com/10395817/137614082-d96b5d24-93e2-4215-b171-ac30f4d35684.png">

This would be a nice to have feature. Power search components are custom-made and extendable.

[kbar](https://kbar.vercel.app) project seems to be an example of good power search implementation with keyboard navigation, back and forth navigation, nested menus, etc.

**Comment by franciscoemanuel** [comment]
Working on it!
